### ISSUE TITLE
[BUG] ModuleNotFoundError when using the latest version

### ISSUE THREAD

**Issue by caiotracera** [Gympass/yoga#434]
**Describe the bug**
I'm getting a `ModuleNotFoundError` when trying to start a new project using the latest version of the `@gympass/yoga`. It seem to be related with the merge of both PRs #431 and #432 .

**To Reproduce**
1. Install the latest version of the lib.
2. The error should be indicated from the moment you import the library into your file.

**Screenshots**
<img width="348" alt="Captura de Tela 2022-02-10 às 16 57 37" src="https://user-images.githubusercontent.com/25802240/153486681-3548b1c7-5fe6-480d-9ebf-d19ac4878cd0.png">

**Desktop (please complete the following information):**
 - OS: macOS Monterey 12.2
 - Browser: Brave

**Additional context**
This [CodeSandbox](https://codesandbox.io/embed/4gug2?fontsize=14&hidenavigation=1) (included in the Yoga documentation) can be used to reproduce the error without the need to start a new project or update an existing one.

Also, I'd like to help to fix this problem.

**Comment by caiotracera** [comment]
I think I've solved the problem by adding the plugin `inline-react-svg` to the `.env.esm` property in the `babel.config.js` at the project root. How can I test it to be sure?

**Comment by caiotracera** [comment]
I've solved the problem by adding the plugin `inline-react-svg` to the property `env.esm` inside the file `babel.config.js` at the project's root folder.

```diff
web: {
       ignore: ['**/native', '**/*.native.js', '**/*.test.jsx'],
       presets: [['@babel/preset-env', { loose: true }], '@babel/preset-react'],
-      plugins: ['import-glob'],
+      plugins: ['inline-react-svg', 'import-glob'],
},
```

From what I understood, Babel doesn't know to transpile any kind of images. So, during the build, he just ignores the import of the SVG. To prevent this behavior, the `inline-react-svg` replaces the SVG import/call to an inline svg creation. 

To be sure that the problem has been solved by this approach, here's what I did:

1. Created a simple CRA app with styled-component called `test-yoga`;
2. Installed `@gympass/yoga` in the latest version to be sure that is broken;
3. Uninstalled `@gympass/yoga` and deleted `node_modules` to prevent cache.
4. Linked both `test-yoga` and my modified version of `@gympass/yoga` using the `npm link` commmand.
5. Got the app running.

<img width="853" alt="Captura de Tela 2022-02-10 às 18 13 06" src="https://user-images.githubusercontent.com/25802240/153497294-8d2099a7-f435-4cc8-886a-a7160c91ef79.png">
 

I'll open the PR with the changes that I made, so we can discuss and maybe release a test version of this package so we can make sure that everything is running.
